packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.workspace.tsx

```diff
@@ -314,6 +320,29 @@ const ImmunizationsForm: React.FC<DefaultPatientWorkspaceProps> = ({
               </ResponsiveWrapper>
             </section>
           )}
+          <section>
+            <ResponsiveWrapper>
+              <Controller
+                name="note"
+                control={control}
+                render={({ field: { onChange, value } }) => (
+                  <div className={styles.row}>
+                    <TextArea
+                      enableCounter
+                      id="note"
+                      invalidText={errors?.note?.message}
+                      labelText={t('note', 'Note')}
+                      helperText={t('addComments', 'Add comments related to this immunization dose')}
```

Comment:
```suggestion
                      helperText={t('addComments', 'Add comments related to this immunization dose')}
```
how about `addImmunizationDoseComments`